Lib/test/test_docxmlrpc.py

```diff
@@ -193,5 +194,27 @@ def test_annotations(self):
             response.read())
 
 
+class XMLRPCDocGeneratorTest(unittest.TestCase):
```

Comment:
IMO you can just add this test to the other class which does very similar `setUp` and `tearDown` since there is only one test for the class.